<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QWidget" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Login - Système de Gestion des Publications de Recherche</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #667eea, stop:1 #764ba2);
}</string>
  </property>
  <layout class="QVBoxLayout" name="mainLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <spacer name="topSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="mainTitle">
     <property name="styleSheet">
      <string notr="true">font-size: 36px;
font-weight: 700;
color: white;
background-color: transparent;
text-align: center;</string>
     </property>
     <property name="text">
      <string>Système de Gestion des Publications</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="spacerBeforeCard">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>30</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="centeringLayout">
     <item>
      <spacer name="horizontalSpacerLeft">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QFrame" name="loginCard">
       <property name="minimumSize">
        <size>
         <width>500</width>
         <height>600</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>500</width>
         <height>600</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QFrame {
    background-color: rgba(255, 255, 255, 0.95);
    border-radius: 25px;
    border: 1px solid rgba(255, 255, 255, 0.2);
}</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::Shape::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Shadow::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="cardLayout">
        <property name="spacing">
         <number>20</number>
        </property>
        <property name="leftMargin">
         <number>50</number>
        </property>
        <property name="topMargin">
         <number>40</number>
        </property>
        <property name="rightMargin">
         <number>50</number>
        </property>
        <property name="bottomMargin">
         <number>40</number>
        </property>
        <item>
         <widget class="QLabel" name="logoPlaceholder">
          <property name="minimumSize">
           <size>
            <width>120</width>
            <height>120</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>120</width>
            <height>120</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #667eea, stop:1 #764ba2);
    border-radius: 60px;
    border: 4px solid white;
    color: white;
    font-weight: bold;
    font-size: 18px;
}</string>
          </property>
          <property name="text">
           <string>LOGO</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="welcomeLabel">
          <property name="styleSheet">
           <string notr="true">font-size: 28px;
font-weight: 600;
color: #2d3748;
background-color: transparent;
text-align: center;</string>
          </property>
          <property name="text">
           <string>Bienvenue</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="fieldSpacer1">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>10</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="instructionLabel">
          <property name="styleSheet">
           <string notr="true">font-size: 16px;
font-weight: 400;
color: #718096;
background-color: transparent;
text-align: center;
margin-bottom: 10px;</string>
          </property>
          <property name="text">
           <string>Connectez-vous pour accéder au système</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="emailEdit">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit {
    background-color: #f7fafc;
    border: 2px solid #e2e8f0;
    border-radius: 12px;
    padding: 0px 15px;
    font-size: 16px;
    color: #2d3748;
}
QLineEdit:focus {
    border: 2px solid #667eea;
    background-color: white;
}</string>
          </property>
          <property name="placeholderText">
           <string>ID</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="passwordEdit">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLineEdit {
    background-color: #f7fafc;
    border: 2px solid #e2e8f0;
    border-radius: 12px;
    padding: 0px 15px;
    font-size: 16px;
    color: #2d3748;
}
QLineEdit:focus {
    border: 2px solid #667eea;
    background-color: white;
}</string>
          </property>
          <property name="echoMode">
           <enum>QLineEdit::EchoMode::Password</enum>
          </property>
          <property name="placeholderText">
           <string>Mot de passe</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="optionsLayout">
          <item>
           <widget class="QCheckBox" name="rememberCheck">
            <property name="styleSheet">
             <string notr="true">QCheckBox {
    color: #4a5568;
    font-size: 14px;
    background-color: transparent;
}</string>
            </property>
            <property name="text">
             <string>Se souvenir de moi</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="forgotPasswordBtn">
            <property name="styleSheet">
             <string notr="true">QPushButton {
    background-color: transparent;
    color: #667eea;
    border: none;
    font-size: 14px;
    font-weight: 500;
    text-decoration: underline;
}
QPushButton:hover {
    color: #764ba2;
}</string>
            </property>
            <property name="text">
             <string>Mot de passe oublié ?</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="loginBtn">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>55</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #667eea, stop:1 #764ba2);
    color: white;
    border: none;
    border-radius: 12px;
    font-size: 18px;
    font-weight: 600;
}
QPushButton:hover {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #5a6fd8, stop:1 #6a4091);
}
QPushButton:pressed {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:0, stop:0 #4a5cc7, stop:1 #5a3579);
}</string>
          </property>
          <property name="text">
           <string>Se connecter</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="signupBtn">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: transparent;
    color: #667eea;
    border: 2px solid #667eea;
    border-radius: 12px;
    font-size: 16px;
    font-weight: 600;
}
QPushButton:hover {
    background-color: rgba(102, 126, 234, 0.1);
    border: 2px solid #764ba2;
    color: #764ba2;
}</string>
          </property>
          <property name="text">
           <string>Créer un nouveau compte</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="orLabel">
          <property name="styleSheet">
           <string notr="true">font-size: 16px;
font-weight: 500;
color: #a0aec0;
background-color: transparent;
text-align: center;
margin: 10px 0px;</string>
          </property>
          <property name="text">
           <string>─ OU ─</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignmentFlag::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="facialAuthBtn">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>55</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
    background-color: white;
    color: #2d3748;
    border: 2px solid #e2e8f0;
    border-radius: 12px;
    font-size: 16px;
    font-weight: 600;
}
QPushButton:hover {
    background-color: #f7fafc;
    border: 2px solid #667eea;
}
QPushButton:pressed {
    background-color: #edf2f7;
}</string>
          </property>
          <property name="text">
           <string>Authentification Faciale</string>
          </property>
          <property name="icon">
           <iconset>
            <normaloff>:/icons/face</normaloff>:/icons/face</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacerRight">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="spacerAfterCard">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Policy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="footerLabel">
     <property name="styleSheet">
      <string notr="true">font-size: 14px;
font-weight: 400;
color: rgba(255, 255, 255, 0.8);
background-color: transparent;
text-align: center;
padding: 20px 0px;</string>
     </property>
     <property name="text">
      <string>© 2024 Système de Gestion des Publications de Recherche - RaMeGaNe</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignmentFlag::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="bottomSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
